python/grass/script/tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+"""Fixtures for Jupyter tests
```

Comment:
Is it specific to Jupyter tests?

Reply:
No, that's a mistake. I tried to mentioned that in the description which was easier at that point than fixing the lines. Fixed now.